Client/components/index.js

```diff
@@ -1,30 +0,0 @@
-import React from 'react';
-import { QueryRenderer, graphql } from 'react-relay';
-
-import environment from './environment';
-import App from './App';
-
-class RendererHelper extends React.Component {
-  render() {
-    return (
-      <QueryRenderer
-        environment={environment}
-        query={graphql`
-          query ChangeName() {
-            name
-          }
-        `}
-        render={({error, props}) => {
-          if (error) {
-            return <div>{error.message}</div>;
```

Comment:
More Udemy course mistakes. This is a function returning React JSX. It HAS to be between parenthesis.